CONTRIBUTING.md

```diff
@@ -40,10 +40,24 @@ We use [standard sphinx docstrings](https://sphinx-rtd-tutorial.readthedocs.io/e
 LinearOperator aims to be fully typed using Python 3.8+
 [type hints](https://www.python.org/dev/peps/pep-0484/).
 We expect any contributions to also use proper type annotations.
-While we currently do not enforce full consistency of these in our continuous integration
-test, you should strive to type check your code locally.
-For this we recommend using [pyre](https://pyre-check.org/).
-
+We are using [jaxtyping](https://github.com/google/jaxtyping) to help us be declarative about the dimension sizes used
+in the LinearOperator methods.
+The use of [jaxtyping](https://github.com/google/jaxtyping)  makes it clearer what the functions are doing algebraically
+and where broadcasting is happening.
+
+These type hints are checked in the unit tests by using
```

Comment:
Yes. The type hints are double checked at runtime by using jaxtyping and typeguard.  The context manager installs a hook to add these checks to the routines called by the test suite. It can then verify that the signatures are working as promised. That is, do the type hints match what *actually* happens when the function is called. You can also do static typing with mypy, flake8 etc since jaxtyping is compatible with that. But, these static checks are less accurate.  So, you can do all the things you did before, this just adds optional extra checks.